docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,73 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_Feynman Zhou, CNCF Ambassador, Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the CLI tool on Aug 22. ORAS v0.14 provides four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
```

Comment:
```suggestion
The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers recently announced v0.14 release for the CLI tool. ORAS v0.14 introduces four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
```

Reply:
No need to commit my suggestions. You can add them directly to the source branch as I don't want to break DCO